src/schema/v2/artist/index.ts

```diff
@@ -860,6 +860,11 @@ export const ArtistType = new GraphQLObjectType<any, ResolverContext>({
         },
       },
       related: Related,
+      recentShow: {
+        type: GraphQLString,
```

Comment:
given no recent show we return "" then this type should be wrapped in a NonNull type, right?

Reply:
Makes sense, I will remove `|| ""` here so we resolve to `null`.